server.js

```diff
@@ -2050,6 +2077,11 @@ if (config.startServer) {
           });
         });
       },
+      async () => {
+        // Now that all migrations have been run, we can start executing any
+        // batched migrations that may have been enqueued by migrations.
+        startBatchedMigrations();
```

Comment:
We'll want to have a config option like `config.runBatchedMigrations` that we can use to disable execution on chunk servers, just like we do with `config.runMigrations`. Or honestly, we could even use the same option - what do you think @mwest1066?

Reply:
I'm slightly tempted to have a separate config option just to give us more control, but I will admit I don't have a specific use case in mind at the moment. So either way is fine by me.

Reply:
`config.runBatchedMigrations` added in b0e45f0.

Reply:
Excellent point.